.github/workflows/rust-clippy.yml

```
@@ -0,0 +1,44 @@
+name: rust-clippy analyze
```

```suggestion
name: CI
```

I don't know the best practices on naming though. But it's just a general CI workflow, isn't it.